packages-native/progress-bar/src/__tests__/ProgressBar.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe("ProgressBar", () => {
 
     it("renders no progress and an error when the value is greater than the maximum", () => {
```

Comment:
Description of the test now should be `it renders progress bar with max value when the value is greater than maximum`

Reply:
Done